app/controllers/mentors_controller.rb

```diff
@@ -68,14 +68,15 @@ def show
 
   def mentor_params
     if params[:mentor].present?
-      params.require(:mentor).permit(
-        :name, :prename, :email, :password, :password_confirmation, :address, :sex,
+      p = [:name, :prename, :email, :password, :password_confirmation, :address, :sex,
         :city, :dob, :phone, :school_id, :field_of_study, :education, :transport,
         :personnel_number, :ects, :term, :absence, :note, :todo, :substitute,
         :filter_by_school_id, :filter_by_meeting_day, :filter_by_coach_id,
-        :exit_kind, :exit_at,
+        :exit_kind, :exit_at, :state,
         :inactive, :photo, schedules_attributes: [:day, :hour, :minute]
-      )
+      ] & current_ability.permitted_attributes(:update, Mentor)
```

Comment:
This + [cancan ability definition](https://github.com/panterch/future_kids/pull/279/files#diff-0fdc2ce2595c4fe479d6a97b404013651e8a0e2d4b02d93c87843b3a30103d9fR77) solves the param protect mechanism.
However if you think it would be better to double secure it with the `param.inspect` thing, I'll add it there too.